<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detector</class>
 <widget class="QWidget" name="Detector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="detectorLayout">
   <item>
    <widget class="QCheckBox" name="cb_displaySegmentation">
     <property name="text">
      <string>Display Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_executeAll">
     <property name="text">
      <string>Execute All</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Num lesion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="detectorResult"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
